files/galaxy/tpv/tool_defaults.yml

```diff
@@ -42,6 +42,17 @@ tools:
               entity.tpv_tags = entity.tpv_tags.combine(
                   TagSetManager(tags=[pulsar_tag])
               )
+      - if: |
+          # This rule will apply when users have non-exsisting remote resources selected or recorded in Galaxy DB
```

Comment:
```suggestion
      - id: removed_remote_resources
        # This rule displays a meaningful error message when users that have selected remote resources that are no longer available (e.g. because they have been removed) attempt to send jobs to them.
        if: |
```